systemtests/volplugin_test.go

```diff
@@ -170,7 +167,7 @@ func (s *systemtestSuite) TestVolpluginRestartMultiMount(c *C) {
 	_, err := s.mon0cmd("sudo truncate -s0 /tmp/volplugin.log")
 	c.Assert(err, IsNil)
 
-	c.Assert(s.createVolume("mon0", "policy1", "test", nil), IsNil)
+	c.Assert(s.createVolume("mon0", "policy1", "test", map[string]string{"unlocked": "true"}), IsNil)
```

Comment:
If its not unlocked, multiple mounts won't be possible which will contradict the test case.